pages/admin/request/create-new.tsx

```diff
@@ -0,0 +1,845 @@
+                  bgColor="white"
+                >
+                  <Text as="h2" mb="20px" textStyle="display-small-semibold" textAlign="left">
+                    {`Search Permit Holder`}
+                  </Text>
+                  <PermitHolderTypeahead onSelect={handleSelectPermitHolder} />
+                </Box>
+                <Box w="100%">
+                  {applicantId &&
+                    (getApplicantLoading ? (
+                      <VStack
+                        h="384px"
+                        w="100%"
+                        justifyContent="center"
+                        border="1px solid"
+                        borderColor="border.secondary"
+                        borderRadius="8px"
+                        bgColor="white"
+                      >
+                        <Spinner
+                          thickness="4px"
+                          speed="0.65s"
+                          emptyColor="gray.200"
+                          color="primary"
+                          size="xl"
+                        />
+                        <Text as="h4" textStyle="display-small-semibold">
+                          Loading Data...
+                        </Text>
+                      </VStack>
+                    ) : (
+                      <SelectedPermitHolderCard applicantId={applicantId} />
+                    ))}
+                </Box>
+              </>
+            ) : (
+              // Create new permit holder
+              <Box
+                w="100%"
+                p="40px"
+                mb="32px"
+                border="1px solid"
+                borderColor="border.secondary"
+                borderRadius="8px"
+                bgColor="white"
+              >
+                <Text as="h2" mb="20px" textStyle="display-small-semibold" textAlign="left">
+                  Creating New Permit Holder
+                </Text>
+                <Text as="h3" mb="8px" textStyle="button-semibold" textAlign="left">
+                  After the request is completed, a new permit holder will be automatically created
+                  in the system.
+                </Text>
+              </Box>
+            )}
+          </VStack>
+        )}
+
+        {/* Forms step */}
+        {step === RequestFlowPageState.SubmittingRequestPage && (
+          <form onSubmit={handleSubmit}>
+            <VStack spacing="32px">
+              <Box
+                w="100%"
+                p="40px"
+                border="1px solid"
+                borderColor="border.secondary"
+                borderRadius="12px"
+                bgColor="white"
+                align="left"
+              >
+                <Text as="h2" textStyle="display-small-semibold" paddingBottom="20px">
+                  {`Permit Holder's Information`}
+                </Text>
+                <PermitHolderInformationForm
+                  permitHolderInformation={{ type: 'NEW', ...permitHolderInformation }}
```

Comment:
```suggestion
                  permitHolderInformation={{ ...permitHolderInformation, type: 'NEW' }}
```
Related to comments in create renewal and replacement application pages